inc/dm_dpp.h

```diff
@@ -29,20 +29,124 @@ class dm_dpp_t {
     ec_data_t    m_dpp_info;
 
 public:
-    int init();
-    ec_data_t *get_dpp_info() { return &m_dpp_info; }
-    int decode(const cJSON *obj, void *parent_id, void* user_info);
-    void encode(cJSON *obj);
+    
+	/**!
+	* @brief Initializes the system or component.
+	*
+	* This function sets up necessary configurations and prepares the system for operation.
+	*
+	* @returns int
+	* @retval 0 on success
+	* @retval -1 on failure
+	*
+	* @note Ensure that all prerequisites are met before calling this function.
+	*/
+	int init();
+    
+	/**!
+	 * @brief Retrieves the DPP information.
+	 *
+	 * This function returns a pointer to the DPP information structure.
+	 *
+	 * @returns A pointer to the `ec_data_t` structure containing DPP information.
+	 *
+	 * @note Ensure that the returned pointer is not modified directly.
+	 */
+	ec_data_t *get_dpp_info() { return &m_dpp_info; }
```

Comment:
For greater clarity with what data is present inside `ec_data_t`
```suggestion
	/**!
	 * @brief Retrieves the DPP Bootstrapping information.
	 *
	 * This function returns a pointer to the DPP Bootstrapping information structure.
	 *
	 * @returns A pointer to the `ec_data_t` structure containing DPP Bootstrapping information.
	 *
	 * @note Ensure that the returned pointer is not modified directly.
	 */
	ec_data_t *get_dpp_info() { return &m_dpp_info; }
```

Reply:
Have committed this change